### DISCUSSION TITLE
How to disable automatic domain assignment to production deployment?

### DISCUSSION THREAD

**Discussion by ahmedbilal** [vercel/community#1831] [Help] [answered]
### Summary

I am using Terraform to create deployment by uploading files. The problem is all my domains/alias gets automatically assigned to the production deployment. I want to explicitly manage the aliases assignment. For example, I want a.example.com assigned to an older production deployment while I want b.example.com and c.example.com assigned to the latest production deployment.

**Comment by amyegan** [comment, marked as answer]
Hi @ahmedbilal! You can set an alias for a specific deployment using [`vercel alias`](https://vercel.com/docs/cli/alias#) with the CLI or [assign an Alias](https://vercel.com/docs/rest-api#endpoints/aliases/assign-an-alias) with the API. You can also include an alias when [creating a new deployment](https://vercel.com/docs/rest-api#endpoints/deployments/create-a-new-deployment) with the API.

**Reply by ahmedbilal** [reply]
@amyegan That's good.
